### DISCUSSION TITLE
How to find out what standard power modes do?

### DISCUSSION THREAD

**Discussion by MateusBMP** [T-Troll/alienfx-tools#216] [Q&A]
In some release, some standard power modes were added on "Fans and Power" screen, at AlienFX Control, but with default names. How can we find out what these power modes do? My bios have 4 power modes, but the AlienFX added 5 power modes and, when i used the Alienware Command Center, witch is now disabled, it have different modes.
Doing basic tests with AlienFX Tools and HWiNFO64 I realize that power modes change power usage, CPU clock, GPU usage and fan controls. However, I couldn't reconcile the different power modes to at least define their names.
From where power modes comes? We managed at least name then properly?

**Comment by T-Troll** [comment]
First mode is always "Manual".
The rest are the same as into AWCC, but it's impossible to retrieve its name (it is not stored into BIOS, and different for different systems).
You should check yourself and rename it into UI.

**Reply by MateusBMP** [reply]
My AWCC doesn't open, only freezes, and shutdown after showing the logo. It's possible to check via terminal which power mode exists? What I mean is if it is possible to retrieve these names via code.

**Reply by T-Troll** [reply]
You can try to RE AWCC, but they have device database online, so it's not simple to intercept it. And this names only present into it.
So just use HWINFO and guess (based on PLs and fan behavior).

**Reply by MateusBMP** [reply]
By the way, I'll use HWINFO to check this. Otherwise, it's interesting. Many hardware changes are made and no one is clear. I'm curious what changes are really being made.
